Add 0.19 release note for PR cloud prepare #1007 (#1192)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

  per service via the `lighthouse.submariner.io/use-clusterset-ip` annotation on the `ServiceExport` or automatically for all services via
  the `enable-clusterset-ip` option on `subctl deploy-broker`. Note that while DNS queries will return the cluster set virtual IP,
  Service Discovery does not route this virtual IP and relies on some external component to do so.

> * New options are added in cloud prepare to support custom vpc in AWS

Reviewer: ```suggestion
* New options were added to `subctl cloud prepare` to support a custom vpc for AWS.
```
